client/jetpack-connect/site-type.js

```diff
@@ -44,13 +44,11 @@ class JetpackSiteType extends Component {
 			<MainWrapper isWide>
 				<div className="jetpack-connect__step">
 					<FormattedHeader
-						headerText={ translate( 'What are we building today?' ) }
-						subHeaderText={ translate(
-							'Choose the best starting point for your site. You can add or change features later.'
-						) }
+						headerText={ translate( 'What type of site are you connecting to Jetpack?' ) }
```

Comment:
Copy from @scottsweb in https://github.com/Automattic/wp-calypso/issues/34328#issuecomment-506295872